wiki/Ranking_Criteria/en.md

```diff
@@ -229,8 +230,8 @@ This category contains explicit allowance statements of concepts and rules that
 - **The following are requirements for background images:**
   - **Minimum width:** 160px
   - **Minimum height:** 120px
-  - **Maximum width:** 2560px
-  - **Maximum height:** 1440px
+  - **Maximum width:** 2560px[^maximum-dimensions]
+  - **Maximum height:** 1440px[^maximum-dimensions]
```

Comment:
These dimensions are larger than I recall them being, kinda scary in terms of the technical requirements but I guess it's what has been in place. Also I don't know who made the storyboard rule about 17m pixels but that's pretty high too. Dangerously.

Reply:
bg image context: [pr](https://github.com/ppy/osu-wiki/pull/2156), [thread 1 (mostly about resolution)](https://osu.ppy.sh/community/forums/topics/837952?n=1), [thread 2 (mostly about file size)](https://osu.ppy.sh/community/forums/topics/853985?n=1)

people use high-fidelity monitors these days so i can definitely understand this one. i'm not sure why a static 1440p image would cause technical concern

sb image context: [pr](https://github.com/ppy/osu-wiki/pull/1225), [thread](https://osu.ppy.sh/community/forums/topics/718100?n=5)

apparently @smoogipoo suggested it, with 4096x4096 being a little under 17m

Reply:
I have no recollection of suggesting this, fwiw. So you can discard anything I suggested and re-evaluate as if I didn't.

Reply:
some of the forum threads mention wanting to get my opinion but either i am very forgetful (not usually the case) or that never happened 😅

Reply:
concerns aside, there are ranked storyboards with images above e.g. 10m pixels fwiw:

- https://osu.ppy.sh/beatmapsets/935708 (4000 x 4000 = 16000000)
- https://osu.ppy.sh/beatmapsets/1134425 (4096 x 2731 = 11186176)
- https://osu.ppy.sh/beatmapsets/1188324 (4500 x 2531 = 11389500)
- https://osu.ppy.sh/beatmapsets/1246880 (2804 x 4096 = 11485184)

there aren't many (probably any) cases where large images are necessary though, because the images can just be split into multiple sprites. [sakuraburst - 「skyshifter vip」](https://osu.ppy.sh/beatmapsets/701482#osu/1637011) for example uses three 1920x1080 px images to create a vertical panorama

Reply:
Multiple smaller images is definitely the preference, to the point I'd consider changing this criterial to reduce the limit to 12m or lower...